toqito/matrix_ops/commutant.py

```diff
@@ -0,0 +1,67 @@
+"""Module for computing the commutant of a set of matrices."""
+
+import numpy as np
+from scipy.linalg import null_space
+
+
+def commutant(A: np.ndarray | list[np.ndarray]) -> list[np.ndarray]:
+    r"""Compute an orthonormal basis (in the Hilbert-Schmidt inner product).
+
+    For the algebra of matrices that commute with each matrix in A.
+
+
+    The commutant condition is given by:
+
+    .. math::
+        A X = X A \\quad \forall A \\in \\mathcal{A}
+
+    This can be rewritten as:
+
+    .. math::
+        (A \\otimes I - I \\otimes A^T) \text{vec}(X) = 0
```

Comment:
Is there a reason that you double use double slashes? For instance `\\otimes` vs. `\otimes`? This is also applicable elsewhere, I think you can just do:
```suggestion
        (A \otimes I - I \otimes A^T) \text{vec}(X) = 0
```